pm_muairss_write/pm_muairss_write.xml

```diff
@@ -65,12 +62,16 @@
             </when>
             <when value="false"/>
         </conditional>
+        <param type="hidden" name="testing" label="Test mode" value="false"/>
     </inputs>
     <outputs>
         <data label="Muonated $structure.name using $params.name" name="muonated_structures" format="zip" from_work_dir="out_zip.zip"/>
-        <data label="File tree (testing only)" name="file_tree" format="txt" from_work_dir="tree.txt" hidden="true"/>
+        <data label="File tree (testing only)" name="file_tree" format="txt" from_work_dir="tree.txt" hidden="true">
+            <filter>(testing == "true")</filter>
+        </data>
     </outputs>
     <tests>
+        <!-- CASTEP with out_folder and castep_param specified, param as txt -->
         <test>
```

Comment:
I you have filters in the outputs it would be good to have expect_num_outputs in the test, to test the filters.